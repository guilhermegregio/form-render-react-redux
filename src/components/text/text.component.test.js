import React from 'react';
import ReactDOM from 'react-dom';

import Text from './text.component';

it('default render text component', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Text id="text" name="text" value="test value" className="text" label="Text" />, div);

    expect(div).toMatchSnapshot();
});
