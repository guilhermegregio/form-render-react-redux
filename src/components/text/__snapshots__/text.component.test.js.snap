// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`default render text component 1`] = `
<div>
  <div
    data-reactroot=""
    class="text-component group text"
  >
    <input
      type="text"
      id="text"
      name="text"
      placeholder=""
      required=""
      value="test value"
    />
    <span
      class="highlight"
    />
    <span
      class="bar"
    />
    <label>
      <!-- react-text: 6 -->
      text
      <!-- /react-text -->
      <!-- react-text: 7 -->
      :
      <!-- /react-text -->
    </label>
  </div>
</div>
`;
